### DISCUSSION TITLE
ImportError: cannot import name 'field_validator' from 'pydantic'. Cant upgrade to Pydantic V2 due to this error.

### DISCUSSION THREAD

**Discussion by nikhilmanuWR** [pydantic/pydantic#9598] [Question] [answered]
ImportError: cannot import name 'field_validator' from 'pydantic' (/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/pydantic/__init__.py). Did you mean: 'root_validator'?

Don't know why this is happening. field_validator is working fine in the IDE itself(as in it is shown in suggestions), this error only comes when we run the app.

PC: macOS
IDE: Pycharm Community Edition

**Comment by uriyyo** [comment, marked as answer]
Hi @nikhilmanuWR,

Are you running your application inside venv? Could you please show output of `pip3 list` command?

**Reply by nikhilmanuWR** [reply]
Hi @uriyyo,

Application was not run inside of venv, even though I thought it was. 
Globally installed pydantic had the version 1.10.x
pip3 list command helped me find this. 

Thank you for answering to my question.

**Reply by uriyyo** [reply]
Great, happy to hear it!
